gnovm/pkg/gnolang/uverse.go

```diff
@@ -848,6 +855,17 @@ func makeUverseNode() {
 		),
 		nil, // results
 		func(m *Machine) {
+			if bm.NativeEnabled {
+				arg0 := m.LastBlock().GetParams1(m.Store)
```

Comment:
duplicate definition with below line.

Reply:
I did it twice to make my benchmark more accurate